.circleci/main-config.yml

```diff
@@ -125,6 +125,9 @@ commands:
             ?(*_)develop)
               DEPLOY_ENV=dev
             ;;
+            DDP-?(*))
+              DEPLOY_ENV=dev
+            ;;
```

Comment:
All the circleci changes are temporary for this branch. Just wanted to test my branch on the server for a realistic test.

Reply:
Or we keep it! Welcome opinions here.

Reply:
I don't think keeping it is a good idea since in case the PR branch has liquibase script changes they might be applied in the wrong order.

Reply:
Ah. Liquibase! That is a good point! Hmm. 
I think the fact that I went through the pain of adding all this stuff points to a need though.
Maybe what we need is a simple way to rollback Liquibase changes if applied?
Or maybe we can figure out way to prevent PRs with Liquibase changes to be deployed in this manner? Maybe.
I am still inclined to leave it in. I think the benefit outweighs the potential problems. But would definitely be good if there is way to avoid the Liquibase problem though.

Reply:
I second @Trimax on this- this violates the contract that what's on `develop` is what's on `dev`. Let's make things easier to deploy locally instead of taking shortcuts for testing.